### DISCUSSION TITLE
Welcome to workflow-kotlin Discussions!

### DISCUSSION THREAD

**Discussion by rjrjr** [square/workflow-kotlin#265] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zach-klippenstein** [comment]
Hello world! I'm a maintainer of this library along with @rjrjr and @steve-the-edwards. I work on the core APIs and the runtime, as well as the Compose integration.

**Comment by mikedtwc** [comment]
Hey folks! I'm just getting started here, checking out Square Workflow Kotlin for potential use on an upcoming project. I love what I'm seeing so far. I was going to post a question about "not seeing the file template options", and notice that I overlooked the link to the instructions on installing them. So I'm good on that now, but expect more questions soon! Thanks :)
